tests/ttnn/unit_tests/operations/test_moreh_fold.py

```diff
@@ -0,0 +1,87 @@
+# SPDX-FileCopyrightText: © 2024 Tenstorrent Inc.
+
+# SPDX-License-Identifier: Apache-2.0
+
+import pytest
+import torch
+import ttnn
+
+from loguru import logger
+from models.utility_functions import comp_allclose, comp_allclose_and_pcc, skip_for_grayskull
```

Comment:
from models.utility_functions import comp_allclose_and_pcc

Reply:
Resolved in latest commit.